yandextank/api/apiworker.py

```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
 """ Provides class to run TankCore from python """
-from yandextank.core import tankcore
 import logging
-from pkg_resources import resource_filename
 import os
 import sys
 import time
 import traceback
+import fnmatch
+from pkg_resources import resource_filename
+
+from ..core import tankcore
```

Comment:
++ @netortik I think you're right.

The answer on stackoverflow is outdated:
> answered Nov 17 '10 at 22:22

Here's the [modern PEP8](https://www.python.org/dev/peps/pep-0008/#imports):

> ...explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose